src/scss-migrate/index.ts

```diff
@@ -34,50 +34,51 @@ export function scssMigrate(_options: Schema): Rule {
-            styleSheetFormat = styleSheetFormat = _options.to;
+            styleSheetFormat = styleSheetFormat = options.to;
           } else {
-            componentSchematics.style = _options.to;
+            componentSchematics.style = options.to;
           }
         } else {
           workspaceSchematics['@schematics/angular:component'] = {
-            "style": _options.to
+            "style": options.to
           };
         }
       } else {
         project.schematics = { ['@schematics/angular:component']: {} };
 
         project.schematics['@schematics/angular:component'] = {
-          "style": _options.to
+          "style": options.to
         };
       }
 
-      const stringifiedWorkspaceConfig = JSON.stringify(workspaceConfig, null, "\t").replace(/styles.css/g, `styles.${_options.to}`);
+      const stringifiedWorkspaceConfig = JSON.stringify(workspaceConfig, null, "\t").replace(/styles.css/g, `styles.${options.to}`);
       const defaultProjectPath = buildDefaultPath(project);
       const lastPosOfPathDelimiter = defaultProjectPath.lastIndexOf('/');
       const srcRoot = defaultProjectPath.substr(0, lastPosOfPathDelimiter + 1);
 
-      tree.exists(`${srcRoot}/styles.${_options.from}`) && tree.rename(
-        `${srcRoot}/styles.${_options.from}`,
-        `${srcRoot}/styles.${_options.to}`
+      tree.exists(`${srcRoot}/styles.${options.from}`) && tree.rename(
+        `${srcRoot}/styles.${options.from}`,
+        `${srcRoot}/styles.${options.to}`
       );
 
-      let filePaths = glob.sync(`.${defaultProjectPath}/**/*.${_options.from}`);
+      let filePaths = glob.sync(`.${defaultProjectPath}/**/*.${options.from}`);
 
-      filePaths = filePaths.length ? filePaths : _options.cssFilesGlob.length ? _options.cssFilesGlob : [];
+      filePaths = filePaths.length ? filePaths : options.cssFilesGlob.length ? options.cssFilesGlob : [];
       filePaths.length && tree.overwrite('/angular.json', stringifiedWorkspaceConfig);
 
       filePaths.forEach((filePath: string) => {
         let relativeComponentClassFileContent: Buffer;
         let filePathNoExtension: string = filePath.substr(0, filePath.lastIndexOf('.'));
         let fileName: string = filePathNoExtension.substr(filePathNoExtension.lastIndexOf('/') + 1, filePathNoExtension.length)
-        let newFilePath: string = `${filePathNoExtension}.${_options.to}`;
+        let newFilePath: string = `${filePathNoExtension}.${options.to}`;
 
 
         // convert file content
-        if (_options.from === 'scss' && _options.to === 'css') {
-          const contentConvertedToCss = scssToCss(filePath);
-          tree.create(newFilePath, contentConvertedToCss);
-          tree.delete(filePath)
+        if (options.from === 'scss' && options.to === 'css') {
+          const data = tree.read(filePath).toString();
+          const result = renderSync({ data });
+          tree.create(newFilePath, result.css.toString());
+          tree.delete(filePath);
```

Comment:
@d-koppenhagen  this seems to delete files without creating their corresponding .css files.
And the content of the deleted files is lost and not copied to the new .css file.
I have reverted this merge.
Please confirm this and create a new PR